### PR TITLE
feat: onboarding flow updates and new flow addition

### PR DESCRIPTION
**Pull request by Calm-Rock**
### Summary

Added flows for 
- ReactJS
-  Other Web Instrumentation - JS

### CODE REVIEW

**Review by ankit01-oss — DISMISSED** [review]

File: frontend/src/container/OnboardingContainer/Modules/APM/Javascript/md-docs/Others/Kubernetes/others-kubernetes-installOtelCollector.md
Comment (makeavish) [inline frontend/src/container/OnboardingContainer/Modules/APM/Javascript/md-docs/Others/Kubernetes/others-kubernetes-installOtelCollector.md:1]: @Calm-Rock Others section for Js should be renamed to Other web/browser instrumentation

File: frontend/src/container/OnboardingContainer/Modules/APM/GoLang/md-docs/MacOsARM64/Recommended/golang-macosarm64-recommended-runApplication.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Once you are done intrumenting your Go Gin application, you can run it using the
  Run this command inside the `otelcol-contrib` directory that you created in the install Otel Collector step
 
 ```bash
-./otelcol-contrib --config ./config.yaml &> otelcol-output.log & echo "$!" > otel-pid
+./otelcol-contrib --config ./config.yaml &> otelcol-output.log & echo "$\!" > otel-pid
```
Comment (makeavish) [inline frontend/src/container/OnboardingContainer/Modules/APM/GoLang/md-docs/MacOsARM64/Recommended/golang-macosarm64-recommended-runApplication.md:9]: Is this intentional change? Asking as I am not sure about this change

File: frontend/src/container/OnboardingContainer/Modules/APM/Javascript/md-docs/Others/Kubernetes/others-kubernetes-installOtelCollector.md
Comment (Calm-Rock) [reply frontend/src/container/OnboardingContainer/Modules/APM/Javascript/md-docs/Others/Kubernetes/others-kubernetes-installOtelCollector.md:1]: Done @makeavish

File: frontend/src/container/OnboardingContainer/Modules/APM/GoLang/md-docs/MacOsARM64/Recommended/golang-macosarm64-recommended-runApplication.md (hunk 1 above)
Comment (Calm-Rock) [reply frontend/src/container/OnboardingContainer/Modules/APM/GoLang/md-docs/MacOsARM64/Recommended/golang-macosarm64-recommended-runApplication.md]: The run statement for MacOs needs to be updated everywhere in the onboarding. 
Will take this up separately. Thankyou for pointing this out!
Have changed it to what it was for now.

**Review by YounixM — APPROVED** [review]

**Pull request merged**